scripts/requirements-compliance.txt

```diff
@@ -2,5 +2,6 @@
 
 # used by ci/check_compliance
 python-magic
+python-magic-bin; sys_platform == "win32"
```

Comment:
Will python-magic-bin work then? I thought Kari had tested it and saw that it was broken.

Reply:
This worked fine for me on Windows 11 + Python 10